draft-normington-jsonpath-latest.txt

```diff
@@ -519,7 +519,7 @@ Internet-Draft   JavaScript Object Notation (JSON) Path   September 2020
    (in base 10 and which may be omitted) separated by colons.
 
    array-slice = [ integer ] ws %x3A ws [ integer ]
-                      [ ws %x3A ws integer ]
+                      [ ws %x3A [ ws integer ] ]
```

Comment:
Should the `ws` be included in the brackets?  Maybe `ws [ %x3A ws [ integer ] ]` is better?

Reply:
It's syntactically equivalent because of the `ws` in the union syntax, but I agree it is more consistent.

Reply:
changed